### PR TITLE
faust: update to 2.37.3

### PR DESCRIPTION
**Pull request by DavideMaggio**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### Local build testing
- I built this PR locally for my native architecture, (ARCH-LIBC)

### CODE REVIEW

**Comment by DavideMaggio** [comment]
...is there anybody out there? :)

File: srcpkgs/faust/template [hunk 1]
```diff
@@ -10,7 +10,7 @@ maintainer="mag <mag-one@autistici.org>"
 license="GPL-2.0-or-later"
 homepage="http://faust.grame.fr/"
```
Comment (paper42) [inline srcpkgs/faust/template:11]:
```suggestion
homepage="https://faust.grame.fr/"
```

**Comment by paper42** [comment]
please squash the commits

**Comment by DavideMaggio** [comment]
...hope I did the right thing this time :|
Thank you @paper42

**Comment by DavideMaggio** [comment]
sorry, this should really be the last one :|

**Comment by paper42** [comment]
> sorry, this should really be the last one :|

Thanks!

**Pull request merged**